LL/lagin.ttl

```diff
@@ -856,33 +805,51 @@
     owl:sameAs <#Eithni>.
 
 <#OengusaMúisc>
-    owl:sameAs <#OengusaMúsc>.
+    owl:sameAs <#OengusaMúsc>;
+    rdfs:comment "Do-bert iarum Oengus ferand do mccaib a ingine. et do-breth leth do Chorpriu mc Con Corb fri cethri mccaib ind arad. & do-breth flaithemnas do.".
+
+<#Corpre-586ad380>
+    a foaf:Person;
+    foaf:title "poeta"@lat, "poet"@eng;
+    irishRel:nomName "Corpre";
+    owl:sameAs <#CorpreCluchechair>;
+    rel:childOf [
+      a foaf:Person;
+      rel:childOf <#OengusaMúisc>;
+      owl:sameAs <#EithniSithbacc>
+    ];
+    rel:relationship <#OengusaMúisc>.
```

Comment:
I'll replace it with foaf:knows, as it really isn't clear what the relationship is meant to be in this instance.